Awesome!! 

Tested with java17. Functions the exact same as it did on j8. Nice work!!

I get this error about it not being able to find unimixins, builds fine if I just provide it myself.
```
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find com.github.LegacyModdingMC.UniMixins:unimixins-all-1.7.10:0.1.14.
     Searched in the following locations:
       - https://maven.minecraftforge.net/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - https://repo.maven.apache.org/maven2/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - https://libraries.minecraft.net/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - https://mvn.falsepattern.com/releases/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - https://mvn.falsepattern.com/cursemaven/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - https://gregtech.overminddl1.com/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - https://mvn.falsepattern.com/sponge/com/github/LegacyModdingMC/UniMixins/unimixins-all-1.7.10/0.1.14/unimixins-all-1.7.10-0.1.14.pom
       - file:/var/tmp/tmp/RightProperGUIScale/build/dirtyArtifacts/unimixins-all-1.7.10-0.1.14-dev.jar
       - file:/var/tmp/tmp/RightProperGUIScale/build/dirtyArtifacts/unimixins-all-1.7.10-dev.jar
     Required by:
         project :
```

--- a/build.gradle
+++ b/build.gradle
@@ -1,4 +1,4 @@
-//version: 1656003793falsepattern72
+//version: 1656003793falsepattern74
 /*
 DO NOT CHANGE THIS FILE!
 
@@ -322,10 +322,14 @@ repositories {
             name = "sponge2"
             url = "https://mvn.falsepattern.com/sponge"
         }
+        maven {
+            name = "jitpack"
+            url = "https://mvn.falsepattern.com/jitpack/"
+        }
     }
 }
 
-def unimixinsVersion = "0.1.14"
+def unimixinsVersion = "0.1.15"
 
 dependencies {
     if(usesMixins.toBoolean()) {

